app/src/main/java/org/simple/clinic/facility/change/FacilityChangeScreen.kt

```diff
@@ -2,7 +2,6 @@ package org.simple.clinic.facility.change
 
```

Comment:
Can you please remove the extra F in the commit message?

Reply:
Whoops, good catch! Will remove.